src/main.ts

```diff
@@ -35,6 +40,20 @@ function run(): void {
       )}`
     );
 
+    core.debug('Switching to the correct Angular Builder');
```

Comment:
suggestion: Improve the debug message `Switching to the correct Angular Builder`. It's not very clear what is happening.